classify/js/src/index.eval.ts

```diff
@@ -0,0 +1,79 @@
+import { Eval } from "braintrust";
+
+import { classifyTitle, initializeOpenAI, loadDataset } from "./utils";
+
+
+const openai = initializeOpenAI();
+const prompt = `
+    You are an editor in a newspaper who helps writers identify the right category for
+    their news articles, by reading the article's title. The category should be one of
+    the following: World, Sports, Business, or Sci-Tech. Reply with one word corresponding
+    to the category`;
+
+const fixedPrompt = `
```

Comment:
make the spacing match between the two prompts -- either both indented or neither